pom.xml

```diff
@@ -620,7 +620,7 @@
     <project.jackson-core-asl.version>1.9.11</project.jackson-core-asl.version>
     <project.jackson-core2.version>2.9.2</project.jackson-core2.version>
     <project.protobuf-java.version>2.6.1</project.protobuf-java.version>
-    <project.guava.version>19.0</project.guava.version>
+    <project.guava.version>24.1-jre</project.guava.version>
```

Comment:
Shoot! I double checked and thankfully it's 20 [1], and 20 supports `throwIfUnchecked`. 

1: Don't know how legit https://stackoverflow.com/questions/36045121/which-version-of-guava-is-compatible-with-java-1-7 is... let me know if you know otherwise!